model_hub/examples/huggingface/language-modeling/README.md

```diff
@@ -0,0 +1,42 @@
+# Language Modeling
+The examples here mirror the [language-modeling examples](https://github.com/huggingface/transformers/tree/master/examples/pytorch/language-modeling) from the original huggingface transformers repo.
+
+You can finetune GPT and GPT-2 with the causal language model (CLM); ALBERT, BERT, DistilBERT, and RoBERTa with the masked language model(MLM); and XLNet with the permutation language model (PLM).
+
+## Files
+* **clm_trial.py**: The [PytorchTrial definition](https://docs.determined.ai/latest/reference/api/pytorch.html#pytorch-trial) for CLM. A few class methods are overwritten and specialized for text classification but otherwise the behavior is the same as the [parent BaseTransformerTrial class](../model_hub/transformers/_trial.py).
```

Comment:
Nit: Technically it's `PyTorchTrial` not `PytorchTrial`. Also don't think you need the "parent" modifier for `BaseTranformerTrial`.